src/IB/IIMethod.cpp

```diff
@@ -638,6 +651,21 @@ IIMethod::postprocessIntegrateData(double /*current_time*/, double /*new_time*/,
     }
     batch_vec_ghost_update(vec_collection_update, INSERT_VALUES, SCATTER_FORWARD);
 
+    if (d_multistep_n_previous_steps > 0)
+    {
+        TBOX_ASSERT(d_multistep_n_previous_steps == 1);
+        // d_U_new_vecs->copy("current", { "old" });
+        // d_U_old_vecs
+
+        for (unsigned part = 0; part < d_num_parts; ++part)
+        {
+            int ierr = VecCopy(d_U_current_vecs[part]->vec(), d_U_old_updated_vecs[part]->vec());
+            IBTK_CHKERRQ(ierr);
+        }
```

Comment:
You can simplify this by using the vector wrapper classes --- please see the corresponding part of `IBFEMethod.cpp`:

https://github.com/IBAMR/IBAMR/blob/12b2f3ec2c38e3ba02d231ded2c81ac6cba09c8b/src/IB/IBFEMethod.cpp#L634-L640